symplyphysics/laws/hydro/input_force_to_area_ratio_equals_to_output.py

```diff
@@ -32,6 +28,21 @@
 
 law = Eq(input_force / input_area, output_force / output_forces_area)
 
+pressure_input = pressure_law.law.rhs.subs({
+    pressure_law.force: input_force,
+    pressure_law.area: input_area
+})
+
+## If the pistons are in equilibrium, then the pressures pressure_input and pressure_output are equal
```

Comment:
To equate pressure on both sides, you should use this law: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/hydro/inner_pressure_of_fluid_is_constant.py
It is not very easy to use though. Here is an example, how to use it: https://github.com/blackyblack/symplyphysics/blob/ea325837f3684750a2fa1b4e68af4d3966a1960c/examples/hydro/pressure_balance_in_u_tube.py#L66